docs-gb/kubernetes/hpa-rps-autoscaling.md

```diff
@@ -246,29 +276,111 @@ spec:
+in the query (`metricsQuery` in the same ConfigMap).
+
+Then, the information provided in the `describedObject` is used within the Prometheus query to
+select the right aggregations of the metric. For the RPS metric we have used to scale the Model
+(and the Server because of the 1-1 mapping), it makes sense to compute the aggregate RPS across
+all the replicas of a given model, so the `describedObject` references a specific Model CR.
 
-In order to ensure similar scaling behaviour between Models and Servers, the number of minReplicas and maxReplicas defined in the HPA, as well as other scaling policies configured should be kept in sync across the HPA for the model and the server.
+However, in the general case, the `describedObject` does not need to be a Model. Any k8s object
+listed in the `resources` section of the Prometheus Adapter ConfigMap may be used. The Prometheus
+label associated with the object kind fills in the `<<.GroupBy>>` template, while the name gets
+used as part of the `<<.LabelMatchers>>`. For example:
+
+* If the described object is `{ kind: Namespace, name: seldon-mesh }`, then the Prometheus
+query template configured in our example would be transformed into:
+
+```
+sum by (namespace) (
+  rate (
+    seldon_model_infer_total{namespace="seldon-mesh"}[1m]
+  )
+)
+```
+
+* If the described object is not a namespace (for example, `{ kind: Pod, name: mlserver-0 }`)
+then the query will be passed the label describing the object, alongside an additional label
+identifying the namespace where the HPA manifest resides in.:
+
+```
+sum by (pod) (
+  rate (
+    seldon_model_infer_total{pod="mlserver-0", namespace="seldon-mesh"}[1m]
+  )
+)
+```
 
-Please note that you **must** use a `target.type` of `AverageValue`. The value given in `averageValue` is the threshold RPS per replica, and the new (scaled) number of replicas is computed by HPA according to the following formula:
+For the `target` of the Object metric you **must** use a `type` of `AverageValue`. The value
+given in `averageValue` represents the per replica RPS scaling threshold of the `scaleTargetRef`
+object (either a Model or a Server in our case), with the target number of replicas being
+computed by HPA according to the following formula:
 
 $$\texttt{targetReplicas} = \frac{\texttt{infer\_rps}}{\texttt{thresholdPerReplicaRPS}}$$
 
 {% hint style="info" %}
-Attempting other target types will not work under the current Seldon Core v2 setup, because they use the number of active Pods associated with the Model CR (i.e. the associated Server pods) in the `targetReplicas` computation. However, this also means that this set of pods becomes "owned" by the Model HPA. Once a pod is owned by a given HPA it is not available for other HPAs to use, so we would no longer be able to scale the Server CRs using HPA.
+Attempting other target types will not work under the current Seldon Core v2 setup, because they
```

Comment:
```suggestion
Attempting other target types does not work under the current Seldon Core 2 setup, because they
```
https://developers.google.com/style/tense